<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>plotter</class>
 <widget class="QWidget" name="plotter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>876</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Plotter</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>861</width>
     <height>531</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <property name="documentMode">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="data_tab">
    <attribute name="title">
     <string>Data</string>
    </attribute>
    <widget class="QFrame" name="data_frame">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>271</width>
       <height>241</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <widget class="QLabel" name="data_frame_lbl">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>231</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Data</string>
      </property>
     </widget>
     <widget class="QLabel" name="data_load_lbl">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>231</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="data_load_btn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Load DE</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="plot_tab">
    <attribute name="title">
     <string>Plot</string>
    </attribute>
    <widget class="QLineEdit" name="xmax_input">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>470</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="ymax_lbl">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>470</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>ymax:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="xmin_input">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>440</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="plot_select_lbl">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Select plot:</string>
     </property>
    </widget>
    <widget class="QLabel" name="title_lbl">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>20</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Title:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="title_input">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>20</y>
       <width>431</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="xmax_lbl">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>470</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>xmax:</string>
     </property>
    </widget>
    <widget class="QFrame" name="plot_frame">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>801</width>
       <height>361</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="ymax_input">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>470</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="ymin_lbl">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>440</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>ymin:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ymin_input">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>440</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="plot_combo">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>181</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Volcano</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="xmin_lbl">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>440</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>xmin:</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
